src/services/storageProvider/hooks/index.js

```diff
@@ -0,0 +1,50 @@
+const CryptoJS = require('crypto-js');
+const { authenticate } = require('@feathersjs/authentication');
+
+const globalHooks = require('../../../hooks');
+
+const encryptSecret = (context) => {
+	if (context.data.secretAccessKey) {
+		context.data.secretAccessKey = CryptoJS.AES
+			.encrypt(context.data.secretAccessKey, process.env.S3_KEY)
+			.toString();
+	}
+	return context;
+};
+
+// const decryptSecret = (context) => {
+// 	if (context.result.data) {
+// 		for (const i in context.result.data) {
+// 			if (context.result.data[i].secretAccessKey) {
+// 				context.result.data[i].secretAccessKey = CryptoJS.AES.decrypt(context.result.data[i].secretAccessKey,
+// 					process.env.S3_KEY).toString(CryptoJS.enc.Utf8);
+// 			}
+// 		}
+// 	} else {
+// 		context.result.secretAccessKey = CryptoJS.AES.decrypt(context.result.secretAccessKey, process.env.S3_KEY)
+// 			.toString(CryptoJS.enc.Utf8);
+// 	}
+//
+// 	return context;
+// };
```

Comment:
Wenn nicht benutzt, dann raus mit dem Code, sonst denkt in Zukunft noch jemand, dass es eine gute Idee wäre, es wieder einzukommentieren :-)